isis/src/control/objs/BundleSolutionInfo/BundleSolutionInfo.h

```diff
@@ -250,7 +257,8 @@ namespace Isis {
       QString             m_name;                        //!< Name of the bundle. Defaults to the id
       QString             m_runTime;                     //!< Run time of the bundle adjustment
       FileName           *m_inputControlNetFileName;     //!< Input control network file name
-      Control            *m_outputControl;               //!< Output control
+      Control *           m_outputControl;               //!< Output control name
```

Comment:
Pointer in ...interesting... location here...

Reply:
Yes.  Originally I tried setting the output control net in jigsaw just to get it's output name out of it.  It's not set when the BundleSolutionInfo object is instantiated in jigsaw, but then I started observing weird behavior in the jigsaw application (the Makefiles for the tests would error out and say that the outputfile was an invalid network).  It seemed somewhat inefficient to instantiate the control network just so I could get it's filename and location to display in bundleout.txt, and there was weirdness going on anyway which I didn't want to chase down for this feature request, so I created the member variable QString m_outputControlName with simple get/set functions.